server/handler/frontend.go

```diff
@@ -61,11 +61,17 @@ func LoadTemplates(c *FilesConfig, basePath string, githubURL string) (templatet
 			return r
 		},
 		"hasRequires": func(requires common.Actors) bool {
-			return !requires.IsEmpty()
+			return len(requires.Users) > 0 || len(requires.Teams) > 0 || len(requires.Organizations) > 0
 		},
 		"getRequires": func(results *common.Result) map[string][]Membership {
 			return getRequires(results, strings.TrimSuffix(githubURL, "/"))
 		},
+		"hasRequiresPermissions": func(requires common.Actors) bool {
+			return len(requires.Permissions) > 0
```

Comment:
I think this should check the result of `requires.GetPermissions()` to make sure it accounts for the legacy properties.

Reply:
Sorry, keep missing these!